internal/repo/htmlToPdf/wkhtmltopdf_test.go

```diff
@@ -0,0 +1,78 @@
+package htmlToPdf
+
+import (
+	"io"
+	"net/http/httptest"
+	"os"
+	"testing"
+)
+
+func TestGetJsonFromHtml(t *testing.T) {
+	htmlToPdf := NewWkHtmlToPdfSvc()
```

Comment:
These would result in failures unless they are executed in an environment where wkthmltopdf is installed.

These test cases need to be skipped when run in github and coverage for this need to be uploaded into the docs directory at project root.

Reply:
done